backend/workers/xnft-wrapper/src/index.ts

```diff
@@ -4,12 +4,36 @@ const RPC = "https://api.devnet.solana.com";
 const PROD_RENDERER_URL =
   "https://unpkg.com/@coral-xyz/react-xnft-dom-renderer@0.1.0-latest.2448/dist/index.js";
 
+const NON_UPGRADED_XNFTS: string[] = [
+  "De4UnwDoaRnPFpy4NXXZCvh4W7JrX5JNcxc6u9iZfZzp",
+  "3i8Av28osHPoWZzWRoU29JBmfSJEcFtJhDzBTLhFG1u6",
+  "9giQG6rC49ZRZVDvi9sVvrYpokYoy12iTkC3CWUtZLpS",
+  "ZtANNvbEeKhfBiFakWbtMcxsYaTRgm6GEtds8PxW2bm",
+  "4QaPNGJFsdqT5cbURcLcVGPQD8GgCpNM6Bmf2p88ex2f",
+  "4AwaNy62XXNhgEbe3Szk9Tb7eEgDcHG3YbpEzdX8DPj5",
+  "FgrUhnsbTYURx1Pyc5D2HDiPaDHzkbH3bv8aDzEmf16x",
+  "HGVjbFZdHuEK1e8MAXte5hG9NquPSig5RobdLvyXvSXG",
+  "8QSN1sG3nWYcTMsCP8BNgjZHdGtvxLU5cbx81KxRrLZq",
+  "9AmGmRkSQYSYAupbdKmr2et8nQkSg5bo8NAG1nmXgY6g",
+  "DFiTps6Xp6hKGE2AWMYveNQ6mqA9TjFa6WJiWG9smHr6",
+  "8tz5Uu7XqiV2YrutBog3VSyNAL4cA3TXdfGnJx2XPPgy",
+  "dShzmjZZeBJhNFXicQKDNkB41NKfaq8jF4vVU7c18pP",
+  "BZ6YRowFVJ69gQRF5nbP4F4uUxukCKS2Lvez5pP6eA75",
+  "Q2H1NKe4NDqsWi8tV1tE8mrxXuGdUXUEghoyffDxhvq",
+  "C6rEseVodzAebN11CHLNLZtYZf9E5XtycbBSDPmYWFb",
+  "HqKfgFm7m4TujMwZ1wsRiEG8kMf5Se6mgK4Y1syQYq69",
+  "AeycivvGzQgT3iEn8pTWuPcazoGSUbKaUn1tLFfmyam4",
+  "9Tqzi3gb4jE3D8Ez79HUTjFfS9f9ES31NjXez6yaffd7",
+];
+
 export default {
   async fetch(request: Request): Promise<Response> {
     const { searchParams, pathname } = new URL(request.url);
 
     let bundle = searchParams.get("bundle");
     let v2 = searchParams.get("v2");
+    //@ts-ignore
+    const xnftAddress: string = searchParams.get("xnftAddress");
```

Comment:
This isn't going to work right because the existing Backpack instances won't send this parameter. Instead we need to have a list of bundle urls.

Reply:
They don't need to though, right? The warning only needs to show up if the user has a new version of the extension but the xnft being asked for is old.
If they have the existing backpack instance the xnft will work as expected for them

Reply:
What if they have an old backpack but a new xNFT?

Reply:
Yeah it'll break then. Assumed we wont be hitting that since we'll deploy first and then ask xnft developers to upgrade.

Should the warning showing up in that case be `Please upgrade your backpack extension` ?

Reply:
Added two different xnfts for both these cases - 
<img width="376" alt="Screenshot 2022-10-16 at 1 50 20 AM" src="https://user-images.githubusercontent.com/8079861/196006111-e23b44f4-cf39-40da-8685-3f2bcba380d9.png">
<img width="374" alt="Screenshot 2022-10-15 at 10 03 35 PM" src="https://user-images.githubusercontent.com/8079861/196006115-afb7e75c-09c2-4051-b248-0a1fe8ed0ba1.png">